.changeset/lucky-tips-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontity/file-settings": minor
+---
+
+README.md full standard
```

Comment:
Please use proper English sentences for the changelogs, ending them with a period. Similar the guidelines we use for comments: https://docs.frontity.org/contributing/code-contribution-guide#code-comments